Sources/Apollo/GraphQLQueryWatcher.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import ApolloCore
```

Comment:
For CocoaPods purposes this needs to be wrapped in an `#if !COOCAPODS` gate because of how we have the lib set up. I'll make a ticket to deal with actually getting the libs to reflect the proper names for coocapods, but for now just slapping the wrapper on should do.

Reply:
Ok, done. We should talk about this separately, but I'm wondering why `ApolloCore` is a separate module, and what should or shouldn't go in there.

Reply:
The short answer is some stuff is used in both the codegen lib and in the main lib, the long answer is that eventually we'd like to [separate the types needed for the codegen from the runtime to reduce what needs to be imported](https://github.com/apollographql/apollo-ios/issues/1047).